schema/src/ProjectInfo.yaml

```diff
@@ -7,41 +7,136 @@ prefixes:
+        range: Location
+        description: The location of the project.
+        slot_uri: schema:location
+        inlined: true
+      projectSize:
+        range: QuantityValue
+        inlined: false
+      environmentType:
+        range: EnvironmentType
+        description: The environment type of the project.
+        multivalued: true
+      region:
+        range: string
+        description: The region associated with the project.
+      bioregion:
+        range: string
+        description: The bioregion associated with the project.
+        multivalued: true
+      biomeType:
+        range: string
+        description: The type of biome associated with the project.
+        multivalued: true
+      watershed:
+        range: string
+        description: The watershed associated with the project.
+      subWatershed:
+        range: string
+        description: The sub-watershed associated with the project.
+      activity:
+        name: activity
+        description: The types of activities being implemented in projects
+        from_schema: https://framework.regen.network/taxonomy/
+        range: ActivityType
+        multivalued: true
+      impact:
+        name: impact
+        description: A project or initiative’s current and potential impact
+        from_schema: https://framework.regen.network/taxonomy/
+        range: ImpactType
+        multivalued: true
+      projectStartDate:
+        slot_uri: xsd:date
+        description: The start date of the project.
+      projectEndDate:
+        slot_uri: xsd:date
+        description: The end date of the project.
+      hasProjectDeveloper:
+        range: Entity
+        description: The organization responsible for owning the project.
+      hasProjectMonitor:
+        range: Entity
+        description: The organization responsible for monitoring the project.
+      hasProjectOperator:
+        range: Entity
+        description: The organization responsible for operating the project.
+      hasProjectOwner:
+        range: Entity
+        description: The organization that owns the project.
+      hasProjectPartner:
+        range: Entity
+        description: The organization that is a partner for the project.
+
+  CreditProjectInfo:
+    is_a: ProjectInfo
+    title: Credit Project Info
+    description: Base class with additional fields for projects that issue credits.
+    slot_usage:
+      projectStartDate:
+        required: true
+      projectEndDate:
+        required: true
+    attributes:
+      hasProjectVerifier:
+        range: Entity
+        description: The organization responsible for owning the project.
+      creditClassVersion:
+        range: string
+        description: The version of the credit class used for the project.
+      marketType:
+        range: MarketTypeTypes
+        description: The type of market for the associated credits.
+        multivalued: true
+
+  Entity:
     slots:
       - name
+      - description
       - url
```

Comment:
does it make sense to use link here?

Reply:
Interesting. Currently in the UI you must be using the entity name as the text for the link? Allowing there to be different link text might change things a little.